src/components/ConfirmDialog.tsx

```diff
@@ -1,4 +1,4 @@
-import { ReactNode } from 'react';
+import { ReactNode, useEffect } from 'react';
```

Comment:
Husk å sjekk for warnings i konsollen :) 
```suggestion
import { ReactNode } from 'react';
```